.github/workflows/build.yml

```diff
@@ -0,0 +1,86 @@
+name: Build Jobs
+
+on: [ push ]
+
+# Build all of the projects and run them against the CASE graph
+
+jobs:
+  dotnet:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Get the code from the repository
+      - name: Get Repo
+        uses: actions/checkout@v3
+
+      # Install the .NET SDK
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3.2.0
+        with:
+          dotnet-version: 6.x
+
+      # Build the project
+      - name: Build
+        run: dotnet build
+        working-directory: ./dotnet/
+
+      # Run the built CASE conversion example
+      - name: Run CASE Conversion
+        run: |
+            dotnet run ../data/geo.json ../output/dotnet.geojson
+            cat ../output/dotnet.geojson
+        working-directory: ./dotnet/
+
+  python:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Get the code from the repository
+      - name: Get Repo
+        uses: actions/checkout@v3
+
+      # Install Python
+      - name: Setup Python
+        uses: actions/setup-python@v4.6.1
+        with:
+          python-version: 3.9
```

Comment:
Should this go into a matrix reflecting the Python versions' maintenance projection?

https://www.python.org/downloads/

If so, I suggest coverage be just Python versions 3.8 and 3.11, not the in-betweens.

Reply:
Addressed in #4 .